src/node.cc

```diff
@@ -1241,6 +1262,26 @@ static ExitCode StartInternal(int argc, char** argv) {
 }
 
 int Start(int argc, char** argv) {
+#ifndef DISABLE_SINGLE_EXECUTABLE_APPLICATION
+  // Repeats argv[0] at position 1 on argv as a replacement for the missing
+  // entry point file path.
+  if (FindSingleExecutableCode() != nullptr) {
```

Comment:
I wonder if this argv replacement could be done somewhere else to keep the overall fn definition of Start clean.